packages/client/src/generation/generateClient.ts

```diff
@@ -728,6 +728,16 @@ function getNodeRuntimeName(engineType: ClientEngineType) {
     return 'library'
   }
 
+  if (engineType === ClientEngineType.Client) {
+    if (!process.env.PRISMA_UNSTABLE_CLIENT_ENGINE_TYPE) {
+      throw new Error(
+        'Unstable Feature: engineType="client" is in a proof of concept phase and not ready to be used publicly yet!',
+      )
+    }
+
+    return 'library' // TODO: eventually use dedicated client runtime bundle
+  }
```

Comment:
ℹ️ Change for ORM-521 to hide client engine behind feature flag env var during `prisma generate`.